src/config/server.rs

```diff
@@ -19,7 +19,7 @@ pub fn get_host() -> String {
 pub fn get_port() -> u16 {
     if let Ok(port) = var(PORT_ENV_VAR) {
         info!("Using {} as the port since {} is set", port, PORT_ENV_VAR);
-        port.parse().expect(format!("The environment variable {PORT_ENV_VAR} contains an invalid port, please fix or delete it").as_str())
+        port.parse().unwrap_or_else(|_| panic!("The environment variable {PORT_ENV_VAR} contains an invalid port, please fix or delete it"))
```

Comment:
This is just applying a suggestion from the linter - normally I'd probably put this in a different commit but ended up being lazy here